### PR TITLE
Modify suffixes and integration tests.

### PR DESCRIPTION
**Pull request by tneymanov**
- Running integration tests right now, but hopefully was diligent enough to make as little errors as possible.

### CODE REVIEW

**Comment by tneymanov** [comment]
Tests passed.

File: gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/large_tests/platinum_partition_without_residual.json [hunk 1]
```diff
@@ -15,51 +15,51 @@
...
         "expected_result": {"num_rows": 7017069}
       },
       {
-        "query": ["SELECT COUNT(0) AS num_rows FROM {DATASET_ID}.{TABLE_ID}___chr01_remaining"],
+        "query": ["SELECT COUNT(0) AS num_rows FROM `{DATASET_ID}.{TABLE_ID}__chr01_remaining`"],
         "expected_result": {"num_rows": 4318727}
       },
       {
-        "query": ["SELECT SUM(start_position) AS sum_start FROM {DATASET_ID}.{TABLE_ID}___chr19_1M"],
+        "query": ["SELECT SUM(start_position) AS sum_start FROM `{DATASET_ID}.{TABLE_ID}__chr19_1M`"],
```
Comment (samanvp) [inline gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/large_tests/platinum_partition_without_residual.json:42]: Let's move queries of each table closer to each other, similar to other tests.

File: gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py [hunk 2]
```diff
@@ -161,23 +161,15 @@ class QueryFormatter(object):
   """
 
   class _QueryMacros(enum.Enum):
-    NUM_ROWS_QUERY = 'SELECT COUNT(0) AS num_rows FROM `{TABLE_NAME}`'
```
[the remark this answers is not in the document]
Comment (tneymanov) [reply gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py:164]: As per offline discussion, punting this work.

Also modified most of tests to point on the default config file.

File: gcp_variant_transforms/libs/bigquery_util.py [hunk 3]
```diff
@@ -373,11 +372,9 @@ def calculate_optimal_partition_size(total_base_pairs):
           partition_size_round_up * (_MAX_BQ_NUM_PARTITIONS - 1))
 
 
-def compose_table_name(base_name, suffix, is_sample=False):
+def compose_table_name(base_name, suffix):
   # type: (str, List[str]) -> str
```
Comment (samanvp) [inline gcp_variant_transforms/libs/bigquery_util.py:376]: Is the second argument type `List[str]`? singular name does not suggest that.

File: gcp_variant_transforms/testing/integration/vcf_to_bq_tests/huge_tests/test_1000_genomes.json [hunk 4]
```diff
@@ -5,22 +5,314 @@
     "input_pattern": "gs://genomics-public-data/1000-genomes/vcf/*.vcf",
     "infer_headers": "True",
     "allow_incompatible_records": "True",
-    "sharding_config_path": "gcp_variant_transforms/testing/data/sharding_configs/integration_homo_sapiens.yaml",
```
Comment (samanvp) [inline gcp_variant_transforms/testing/integration/vcf_to_bq_tests/huge_tests/test_1000_genomes.json:8]: Please remove `integration_homo_sapiens.yaml` from the repo.

File: gcp_variant_transforms/libs/bigquery_util.py (hunk 3 above)
Comment (tneymanov) [reply gcp_variant_transforms/libs/bigquery_util.py]: Nice catch, modified.

File: gcp_variant_transforms/testing/integration/vcf_to_bq_tests/huge_tests/test_1000_genomes.json (hunk 4 above)
Comment (tneymanov) [reply gcp_variant_transforms/testing/integration/vcf_to_bq_tests/huge_tests/test_1000_genomes.json:8]: Done.

**Pull request merged**